docs/hazmat/primitives/key-derivation-functions.rst

```diff
@@ -372,6 +478,7 @@ Interface
 
 
 .. _`NIST SP 800-132`: http://csrc.nist.gov/publications/nistpubs/800-132/nist-sp800-132.pdf
+.. _`NIST SP 800-56Ar2`: http://dx.doi.org/10.6028/NIST.SP.800-56Ar2
```

Comment:
This URL actually `303` redirects me to http://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-56Ar2.pdf. Is there any reason not to link to this URL instead?